Resources/Private/Forms/Contact.form.yaml

```diff
@@ -8,13 +8,10 @@ finishers:
         identifier: EmailToReceiver
         options:
             subject: '{subject}'
-            recipientAddress: ''
-            recipientName: ''
             senderAddress: '{email}'
             senderName: '{fullname}'
-            replyToAddress: '{email}'
-            carbonCopyAddress: ''
-            blindCarbonCopyAddress: ''
```

Comment:
These have been dropped without replacement?

Reply:
CC and BCC are optional and should be skipped if no useful static value can be set.